src/main/java/Problem1/ReverseStack.java

```diff
@@ -4,6 +4,30 @@
 
 public class ReverseStack {
     public static void reverseStack(Stack<Integer> stack) {
-        // homework
+
+        if (stack.size() != 0)
+        {
+            int i = stack.pop();
+
+            reverseStack(stack);
+            reverseOrder(stack, i);
+        }
+    }
+
+
+    public static void reverseOrder(Stack<Integer> stack, int i) {
```

Comment:
nit picking: the name of this function doesn't really match what it does. this functions basically insert an element at the bottom of the stack. rename it to insertAtBottom would be much clearer.